Automattic/newspack-listings
# feat: display options for curated lists

### All Submissions:






### Changes proposed in this Pull Request:



Enables all display/design options for Curated List block. The styles here are placeholder, mostly to demonstrate how they might work.

Also fixes inheritance of Curated List block attributes to Listing blocks.

### How to test the changes in this Pull Request:

1. Create a curated list block and add several list items.
2. Change the display options for the block and observe that the settings are reflected for all list items in both editor and front-end views.

Display options to test in this PR:

- **Show list item numbers:** toggles display of numbers alongside listings
- **Show featured image:** toggles display of featured image
- **Show featured image caption** (only available if featured image is enabled): toggles display of featured image caption
- **Featured image position** (only available if featured image is enabled): sets position of featured image to top, left, or right
- **Mobile stack** (only available if featured image position is **left** or **right**): if ON, featured image will stack on top of post content at smaller viewports.
- **Show author:** toggles display of listing post authors
- **Show excerpt:** toggles display of listing post excerpts
- **Type scale:** sets font size of listing title, author, and excerpt
- **Text color:** sets font color of listing title, author, and excerpt

### Other information:
